### PR TITLE
Add support for Bintray gradle uploads

### PR DESCRIPTION
**Pull request by rogerhu**

### CODE REVIEW

File: Parse/build.gradle [hunk 1]
```diff
@@ -2,10 +2,17 @@ import com.android.builder.core.BuilderConstants
 
 apply plugin: 'com.android.library'
 apply plugin: 'com.github.kt3k.coveralls'
+apply plugin: 'com.jfrog.bintray'
 
 group = 'com.parse'
 version = '1.14.1-SNAPSHOT'
 
+ext {
+    desc = 'A library that gives you access to the powerful Parse cloud platform from your Android app.'
```
Comment (flovilmart) [inline Parse/build.gradle]: can we have fully named variables please?

File: Parse/build.gradle [hunk 2]
```diff
@@ -74,21 +81,24 @@ android.libraryVariants.all { variant ->
     def javadoc = task("javadoc${variant.name.capitalize()}", type: Javadoc) {
         description "Generates Javadoc for $variant.name."
         source = variant.javaCompile.source
-        ext.androidJar = "${android.sdkDirectory}/platforms/${android.compileSdkVersion}/android.jar"
+        ext.androidJar =
```
Comment (flovilmart) [inline Parse/build.gradle]: revert that chnge

File: Parse/build.gradle [hunk 3]
```diff
@@ -74,21 +81,24 @@ android.libraryVariants.all { variant ->
...
+                "${android.sdkDirectory}/platforms/${android.compileSdkVersion}/android.jar"
         classpath = files(variant.javaCompile.classpath.files) + files(ext.androidJar)
 
         options.docletpath = [rootProject.file("./gradle/ExcludeDoclet.jar")]
         options.doclet = "me.grantland.doclet.ExcludeDoclet"
 
-        options.linksOffline("http://d.android.com/reference", "${android.sdkDirectory}/docs/reference")
+        options.linksOffline("http://d.android.com/reference",
```
Comment (flovilmart) [inline Parse/build.gradle]: revert that too

File: Parse/build.gradle [hunk 4]
```diff
@@ -74,21 +81,24 @@ android.libraryVariants.all { variant ->
...
+                "${android.sdkDirectory}/docs/reference")
         options.links("http://boltsframework.github.io/docs/android/")
 
         exclude '**/BuildConfig.java'
         exclude '**/R.java'
         exclude '**/internal/**'
     }
 
-    def javadocJar = task("javadocJar${variant.name.capitalize()}", type: Jar, dependsOn: "javadoc${variant.name.capitalize()}") {
+    def javadocJar = task("javadocJar${variant.name.capitalize()}", type: Jar,
```
Comment (flovilmart) [inline Parse/build.gradle]: revert

File: Parse/build.gradle [hunk 5]
```diff
@@ -105,8 +115,10 @@ apply plugin: 'maven'
 apply plugin: 'signing'
 
 def isSnapshot = version.endsWith('-SNAPSHOT')
-def ossrhUsername = hasProperty('NEXUS_USERNAME') ? NEXUS_USERNAME : System.getenv('CI_NEXUS_USERNAME')
-def ossrhPassword = hasProperty('NEXUS_PASSWORD') ? NEXUS_PASSWORD : System.getenv('CI_NEXUS_PASSWORD')
+def ossrhUsername =
```
Comment (flovilmart) [inline Parse/build.gradle]: revert

File: build.gradle [hunk 6]
```diff
@@ -10,6 +9,12 @@ buildscript {
     }
 }
 
+plugins {
+    id "com.jfrog.bintray" version "1.7.3"
+}
+
+apply plugin: 'com.github.ben-manes.versions'
```
Comment (flovilmart) [inline build.gradle]: what's that plugin?

Comment (rogerhu) [inline build.gradle]: https://github.com/ben-manes/gradle-versions-plugin -> shows what versions you're using vs. what the latest is

Comment (rogerhu) [inline build.gradle]:
```
------------------------------------------------------------
: Project Dependency Updates (report to plain text file)
------------------------------------------------------------

The following dependencies are using the latest release version:
 - com.parse.bolts:bolts-tasks:1.4.0
 - org.kt3k.gradle.plugin:coveralls-gradle-plugin:2.8.1
 - com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3
 - com.github.ben-manes:gradle-versions-plugin:0.14.0
 - org.skyscreamer:jsonassert:1.4.0
 - com.squareup.okhttp3:mockwebserver:3.6.0
 - com.squareup.okhttp3:okhttp:3.6.0
 - com.facebook.stetho:stetho:1.4.2

The following dependencies have later release versions:
 - com.android.support:appcompat-v7 [25.2.0 -> 25.3.0]
 - com.android.tools.build:gradle [2.3.0 -> 2.5.0-alpha-preview-01]
 - org.mockito:mockito-core [1.10.19 -> 2.7.17]
 - org.robolectric:robolectric [3.3 -> 3.3.1]

Generated report file build/dependencyUpdates/report.txt
```

File: Parse/build.gradle (hunk 1 above)
Comment (rogerhu) [inline Parse/build.gradle]: fixed

File: Parse/build.gradle (hunk 2 above)
Comment (rogerhu) [inline Parse/build.gradle]: done

File: Parse/build.gradle (hunk 3 above)
Comment (rogerhu) [inline Parse/build.gradle]: done

File: Parse/build.gradle (hunk 4 above)
Comment (rogerhu) [inline Parse/build.gradle]: done

File: Parse/build.gradle (hunk 5 above)
Comment (rogerhu) [inline Parse/build.gradle]: done

**Comment by rogerhu** [comment]
Leaving alone for now.

**Pull request closed without merging**